syntaxes/rnc.tmLanguage.json

```diff
@@ -0,0 +1,86 @@
+{

```

Comment:
This RelaxNG TextMate is the result of JSON convertion from the XML TextMate https://github.com/lilyball/textmate-bundles/blob/master/Review/Bundles/RelaxNG.tmbundle/Syntaxes/RelaxNG%20Compact.tmLanguage